ODIN_II/regression_test/benchmark/task/keywords/begin_end/simulation_result.json

```diff
@@ -0,0 +1,326 @@
+        "simulation_time(ms)": 2.7,
+        "test_coverage(%)": 100,
+        "Latch Drivers": 0,
+        "Pi": 11,
+        "Po": 3,
+        "logic element": 16,
+        "latch": 0,
+        "Adder": -1,
+        "Multiplier": -1,
+        "Memory": -1,
+        "Hard Ip": -1,
+        "generic logic size": -1,
+        "Longest Path": 7,
+        "Average Path": 5,
+        "Estimated LUTs": 16,
+        "Total Node": 16
+    },
+    "begin_end/negedge/k6_frac_N10_frac_chain_mem32K_40nm": {
+        "test_name": "begin_end/negedge/k6_frac_N10_frac_chain_mem32K_40nm",
+        "architecture": "k6_frac_N10_frac_chain_mem32K_40nm.xml",
+        "blif": "negedge.blif",
+        "exit": 0,
+        "errors": [],
+        "warnings": [],
+        "expectation": [],
+        "max_rss(MiB)": 36.1,
+        "exec_time(ms)": 84.9,
+        "simulation_time(ms)": 1.4,
+        "test_coverage(%)": 100,
+        "Latch Drivers": 1,
+        "Pi": 1,
+        "Po": 2,
+        "logic element": 5,
+        "latch": 1,
+        "Adder": -1,
+        "Multiplier": -1,
+        "Memory": -1,
+        "Hard Ip": -1,
+        "generic logic size": 4,
+        "Longest Path": 6,
+        "Average Path": 4,
+        "Estimated LUTs": 5,
+        "Total Node": 7
+    },
+    "begin_end/negedge/k6_N10_40nm": {
+        "test_name": "begin_end/negedge/k6_N10_40nm",
+        "architecture": "k6_N10_40nm.xml",
+        "blif": "negedge.blif",
+        "exit": 0,
+        "errors": [],
+        "warnings": [],
+        "expectation": [],
+        "max_rss(MiB)": 18.2,
+        "exec_time(ms)": 13.2,
+        "simulation_time(ms)": 1.5,
+        "test_coverage(%)": 100,
+        "Latch Drivers": 1,
+        "Pi": 1,
+        "Po": 2,
+        "logic element": 5,
+        "latch": 1,
+        "Adder": -1,
+        "Multiplier": -1,
+        "Memory": -1,
+        "Hard Ip": -1,
+        "generic logic size": 6,
+        "Longest Path": 6,
+        "Average Path": 4,
+        "Estimated LUTs": 5,
+        "Total Node": 7
+    },
+    "begin_end/negedge/k6_N10_mem32K_40nm": {
+        "test_name": "begin_end/negedge/k6_N10_mem32K_40nm",
+        "architecture": "k6_N10_mem32K_40nm.xml",
+        "blif": "negedge.blif",
+        "exit": 0,
+        "errors": [],
+        "warnings": [],
+        "expectation": [],
+        "max_rss(MiB)": 35.1,
+        "exec_time(ms)": 84.8,
+        "simulation_time(ms)": 1.4,
+        "test_coverage(%)": 100,
+        "Latch Drivers": 1,
+        "Pi": 1,
+        "Po": 2,
+        "logic element": 5,
+        "latch": 1,
+        "Adder": -1,
+        "Multiplier": -1,
+        "Memory": -1,
+        "Hard Ip": -1,
+        "generic logic size": 6,
+        "Longest Path": 6,
+        "Average Path": 4,
+        "Estimated LUTs": 5,
+        "Total Node": 7
+    },
+    "begin_end/negedge/no_arch": {
+        "test_name": "begin_end/negedge/no_arch",
+        "architecture": "n/a",
+        "blif": "negedge.blif",
```

Comment:
I don't think this ever worked, can you check the vector for this one and make sure it triggers on a negedge?

Reply:
I did and I was pretty sure it does

Reply:
awesome! I fixed it last year (part of the multiclock issue from huawei) just double checking.. I though that was still an issue, or its the question bellow...

Reply:
It's the question below I think